compatibility_lib/compatibility_lib/utils.py

```diff
@@ -86,8 +86,8 @@ def __init__(self, py_version=None, checker=None, store=None):
         self.checker = checker
         self.store = store
 
-    def _get_from_bigquery(self, package_name):
-        """Gets the package dependency info from bigquery
+    def _get_from_compatibility_store(self, package_name):
```

Comment:
`_get_from_cloud_sql` will describe it more accurate.

Reply:
done